backend/src/cache/deployer/webhook-create-signed-cert.sh

```diff
@@ -115,7 +115,7 @@ spec:
   usages:
   - digital signature
   - key encipherment
-  - server auth
+  - client auth
```

Comment:
I think the cache server needs the CSR to have server auth, otherwise it has the following errors in the cache server.
```
2022/02/07 23:12:31 http: TLS handshake error from 172.30.4.65:56336: remote error: tls: bad certificate
```